django/forms/fields.py

```diff
@@ -204,6 +204,19 @@ def has_changed(self, initial, data):
         data_value = data if data is not None else ''
         return initial_value != data_value
 
+    def get_bound_field(self, form, name):
```

Comment:
Seems fine to rename everywhere, but it seems out of the scope of this PR to touch `BoundField.__init__`.